ansible/roles/test/tasks/arp_clear.yml

```diff
@@ -20,26 +20,16 @@
       interface_facts:
       when: ansible_interface_facts is not defined
 
-    - name: Read the current interfaces status
-      shell: show interfaces status | head
-      register: show_interface_status
+    - name: Store the indices for the ports that operationally 'up'
+#      set_fact: candidate_ports="{{ candidate_ports|default([]) + [minigraph_port_indices[item.device]] }}"
```

Comment:
do we want to commit a commented out set_fact?  should we just delete this?

Reply:
You're right, it doesn't need to be there.